nfsn-cron.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
-set -o errexit
+set -eu
```

Comment:
I prefer to use long names for shell options, for clarity. This is a good idea, though.